src/Queue.ts

```diff
@@ -182,6 +188,31 @@ export class Queue {
     stop() {
         this.isActive = false;
     }
+
+    /**
+     * cancel running job
+     */
+    cancelJob(jobId: string, exception?: Error) {
+      const promise = this.runningJobPromises[jobId]
+      if (promise && typeof promise[CANCEL] === 'function') {
+        promise[CANCEL](exception || new Error(`canceled`))
+      } else if(!promise[CANCEL]){
```

Comment:
I am sorry, i did a mistake. Can you revert this to throwing an error or a warn log? I didn't recognized that this would be an inconsitent behaviour, since the job just fail on cancel and isn't removed.

Reply:
Ohkay cool

Reply:
resolved